kctf-operator/pkg/controller/challenge/challenge_default.go

```diff
@@ -16,8 +16,7 @@ func PersistentVolumeClaimsDefault(challenge *kctfv1alpha1.Challenge) *corev1.Pe
 		Items: []corev1.PersistentVolumeClaim{
 			corev1.PersistentVolumeClaim{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      challenge.Name,
-					Namespace: challenge.Namespace,
+					Name: challenge.Name + "-sessions",
```

Comment:
"sessions" is specific to the web challenges. Maybe a more generic name could be "-storage"?

Reply:
lets ask the user to always ask for the volumes instead of having a default, so they dont introduce vulns when creating subdirectories.